.github/workflows/ci.yml

```diff
@@ -19,25 +19,20 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - name: Install Node
-        uses: actions/setup-node@v3
+      - uses: wyvox/action-setup-pnpm@v2
```

Comment:
minor, this was removed from addon blueprint recently https://github.com/embroider-build/addon-blueprint/blob/main/files/.github/workflows/ci.yml#L22